lib/wcc/contentful/model/page.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class WCC::Contentful::Model::Page < WCC::Contentful::Model
+  validate_field :title, :String
+  validate_field :slug, :String
+  validate_field :subpages, :Array, link_to: %w[page]
+  validate_field :sections, :Array
```

Comment:
This can also have a `link_to` which accepts a regexp.  Example from the `model_validators_spec.rb`:

```ruby
    it 'should validate links to content type by regexp' do
      _my_class =
        Class.new(base_class('page')) do
          validate_field :sections, :Array, link_to: /^section/
        end

      # act
      result = WCC::Contentful::Model.schema.call(transformed)

      # assert
      expect(result).to be_success
    end
```

Then the model validators will ensure that the `sections` array accepts only content types beginning with the string `section`.